crates/consensus/src/validation.rs

```diff
@@ -46,21 +46,23 @@ pub fn validate_header_standalone(
 
     // EIP-3675: Upgrade consensus to Proof-of-Stake:
     // https://eips.ethereum.org/EIPS/eip-3675#replacing-difficulty-with-0
-    if Some(header.number) >= chain_spec.paris_status().block_number() {
-        if header.difficulty != U256::ZERO {
-            return Err(Error::TheMergeDifficultyIsNotZero)
-        }
+    if let ParisStatus::Supported { block: Some(block), .. } = chain_spec.paris_status() {
```

Comment:
note this is the change @onbjerg has fixed here I believe https://github.com/paradigmxyz/reth/pull/1128/files#diff-19183879c69f09559b20be39d089113af8aaa35cce426c71df0b6c15e082e771R52-R53 so would be good for you guys to coordinate on this. maybe drop from this PR.